website/docs/enterprise/pipelines/promoting-applications.mdx

```diff
@@ -127,13 +127,33 @@ Now as soon as the `HelmRelease` on the first environment defined in the pipelin
 
 ## Pull request
 
-This section covers adding a promotion by pull request (PR) strategy, so that whenever the application defined in a pipeline is upgraded in one of the pipeline's environments, a GitHub PR is created that updates the manifest file setting the application version in the next environment.
+This section covers adding a promotion by pull request (PR) strategy, so that whenever the application defined in a pipeline
+is upgraded in one of the pipeline's environments, a PR is created that updates the manifest file setting the application version in the next environment.
 
-The dynamic nature of GitOps deployments requires you to assist Weave GitOps a little with information on which repository hosts the manifest files, how to authenticate with the repository and the GitHub API, and which file hosts the version definition for each environment.
+The dynamic nature of GitOps deployments requires you to assist Weave GitOps a little with information on which repository hosts the manifest files,
+how to authenticate with the repository and the git provider API, and which file hosts the version definition for each environment.
+
+### Supported Git Providers
+You could currently open pull request for the following git providers
+
+- [Github](https://github.com/)
+- [Gitlab](https://gitlab.com/)
+
+Select your git provider via `.spec.promotion.pull-request.type`. For example, for `gitlab` it would look similar to:
+
+```yaml
+  promotion:
+    pull-request:
+      type: "gitlab"
+```
+
+More info in the [spec](../spec/v1alpha1/pipeline/#pipeline)
 
 ### Create credentials secret
 
-In order to authenticate with the GitHub repository hosting the manifests, you need to create a Secret with credentials used for cloning. The contents of this secret are the same you use for [GitRepositories](https://fluxcd.io/flux/components/source/gitrepositories/#secret-reference). In addition to these you need to include a `token` field containing a GitHub access token that is used to authenticate with the GitHub API when creating a pull request. Here is an example Secret:
+In order to authenticate with the git provider repository hosting the manifests, you need to create a Secret with credentials used for cloning.
+The contents of this secret are the same you use for [GitRepositories](https://fluxcd.io/flux/components/source/gitrepositories/#secret-reference).
```

Comment:
Whilst we're here, this is the only line where secret is not capitalized, we should be consistent

Reply:
updated